constants/categories.ts

```diff
@@ -4,97 +4,145 @@ const categories: { [route: string]: CopyHolder } = {
   '/food': {
     english: {
       category: 'Food',
+      description:
```

Comment:
Do we need welcome/here are resources copy? it seems a bit extraneous and unnecessary.